Cod. orient. 271a nv

nafisa-valieva opened the pull request:
I have updated the record for the exhibition. I have not added incipit or explicit, since the text is available in LIT id. Let me know if you want me to add the text.

karljonaskarlsson commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
-                                    <locus from="64" to="71"/>
-                                  </item>
+                <collation>
+                  <signatures> Quire marks on the upper left corner of the first leaf of each quire
```
If you want to and have the information available, it is possible to encode the quire numbers also within the individual quires. I think it looks good in the visualization :)

nafisa-valieva replied:
It means to change Eliana´s work, but I think it is ok to do so.

karljonaskarlsson commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+                  </dimensions>
+                  <ab type="ruling" subtype="pattern">1-1-11/0/0/C</ab>
+                  <ab type="pricking" subtype="pattern"> The layout corresponds to the pattern I
+                    defined by D. Nosnitsin: the text is written above the first ruled line, the
```
Maybe better to refer to Denis's publication than just to his name?

CarstenHoffmannMarburg replied:
"pattern I defined" is confusing, because "I" can be read as "me". Better: "The layout corresponds to the pattern 1, defined by D. Nosnitsin."

karljonaskarlsson replied:
But if Denis uses Latin numbers, I think it's better to keep them. However, maybe "Pattern I", with a capitalized "p", would make it clear? It could also be italicized.

CarstenHoffmannMarburg replied:
I see, that a comma in this place is grammatically incorrect. It is better to write this way:
The layout corresponds to pattern I as defined by D. Nosnitsin:

karljonaskarlsson commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+                  </dimensions>
+                  <ab type="ruling" subtype="pattern">1-1-11/0/0/C</ab>
+                  <ab type="pricking" subtype="pattern"> The layout corresponds to the pattern I
+                    defined by D. Nosnitsin: the text is written above the first ruled line, the
```
Again here

karljonaskarlsson commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+              <handNote script="Ethiopic" xml:id="h1">
+                <date notBefore="1700" notAfter="1800"/>
+                <seg type="ink">18th cent. Black, red</seg>
+                <note>According to Charles, ´It was written in the 18th cent. Is was possibly
```
Here it would also be good to refer to the publication

karljonaskarlsson commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+            <additions>
+              <list>
+                <item xml:id="a1">
+                  <desc>Donation note from J. M. Flad to his beloved brother Haussmann.</desc>
```
This, I suppose, is PRS4133Flad. Maybe one could add the reference?

nafisa-valieva replied:
It is an archival record, as far as I could see, it has not been published yet. I have discussed with Denis what would be a solution and came up with the record you have seen. If you want me to refer to this archival record which needs to be treated as a manuscript, I will do it, but I would be grateful for an example.

eu-genia commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+                  <locus target="#1r"/>
+                </item>
+                <item xml:id="a3">
+                  <desc>Small stripe of paper: "Handschriften Sammlung Freihr. a.d. v. Westenholz
```
Mark up all names

CarstenHoffmannMarburg commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+                    in the outer. </ab>
+                </layout>
+                <layout columns="2" writtenLines="26 27">
+                  <note>Data on dimension are taken from <locus target="#68r"/>.</note>
```
dimensions

CarstenHoffmannMarburg commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -1,316 +1,342 @@
+                    <dim type="intercolumn">10</dim>
+                  </dimensions>
+                  <ab type="ruling" subtype="pattern">1-1-11/0/0/C</ab>
+                  <ab type="pricking" subtype="pattern"> The layout corresponds to the pattern I
```
1 and comma, as above

nafisa-valieva replied:
I do not understand what you mean!

karljonaskarlsson commented:
@nafisa-valieva , you have still not implemented the changes for some of the things you "thumbs-upped", it seems! Just so you don't forget :)

nafisa-valieva commented:
> @nafisa-valieva , you have still not implemented the changes for some of the things you "thumbs-upped", it seems! Just so you don't forget :)

yes yes! I have implemented the changes, but not pushed them yet because there are things I am working on.

nafisa-valieva commented:
@karljonaskarlsson @eu-genia @CarstenHoffmannMarburg, i believe I have addressed all your comments and would be happy now, in case I have not missed anything, to have you approve.

eu-genia reviewed (approved):
thank you

nafisa-valieva commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -215,18 +219,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                <item xml:id="a2">
-                  <desc type="StampExlibris">Signature written in pencil: "Cod. Orient. 271a = 130
+                <item xml:id="e1">
+                  <desc>Signature written in pencil: "Cod. Orient. 271a = 130
```
> Why did you delete StampExlibris here?

I have spoken with Dorothea, these types of additions are rather element, elements do not need ´type´

nafisa-valieva commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -215,18 +219,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                <item xml:id="a3">
-                  <desc type="Unclear">Note written in pencil: "1905 5388"</desc>
+                <item xml:id="e2">
+                  <desc>Note written in pencil: "1905 5388"</desc>
```
same as above.

nafisa-valieva commented on Hamburg/Stabi/SHOr271a.xml:
```diff
@@ -187,45 +187,48 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                  in the 18th cent. Is was possibly written by the same scribe as Ms.o
-                  (=Ms.orient.491 [lies: 499] in the British Museum).`</note>
+                  in the 18th cent. Is was possibly written by the same scribe as Ms.o (=<ref
+                    type="mss" corresp="BLorient491">.orient.491 [lies: 499] in the British
```
to the first comment: I do not know, I am just quoting the quoted comment. I understand your point, but I will think what to do.
